### DISCUSSION TITLE
combine filter with exact match switch

### DISCUSSION THREAD

**Discussion by eadmaster** [junegunn/fzf#3694] [Q&A] [answered]
if i understand correctly, this shouldn't match any string:

````
$ echo 'first_second_third_convert
first_second_third_query
first.second
first-second' | fzf -e -f "s c"

first_second_third_convert
first.second
first-second
first_second_third_query
````

Is it possible to have fzf matching whole words only?
(no matches in the above example)

**Comment by junegunn** [comment, marked as answer]
In that case `s` and `c` are treated as two separate search terms.

https://github.com/junegunn/fzf?tab=readme-ov-file#search-syntax

You can either escape the space (i.e. `s\ c`) or turn off the extended search mode with `+x` (or `--no-extended`) which I don't recommend.
